GroundDBMemoryPool/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ project(GroundDBMemoryPool)
 
 include_directories(${CMAKE_SOURCE_DIR}/src/include)
 include_directories(/usr/include/rdma)
+# todo: We will need to use macros and types in OpenAurora
+# include_directories(${CMAKE_SOURCE_DIR}/../OpenAurora/include)
```

Comment:
Does line 7 work?

Reply:
No, when I tried to include it, the compilation did not success. I think it was because we lacked the compilation options that was originally set by OpenAurora's CMakeLists.txt

Reply:
Is it possible to extract the headers we actually need to use from OpenAurora and put it in GroundDB-MemPool

Reply:
If I uncomments line7, the headers we need recursively include other headers and leads to compilational error. Or is there another way to do it?

Reply:
I think only a handful of struct definitions are needed for GroudDB.  Can you copy those macros and structs, and extract them to a separate header file in GroundDBMemoryPool->src->include? It can be done in a separate PR tho